lib/platform/git/storage.ts

```diff
@@ -308,19 +308,16 @@ export class Storage {
   async getFileList(): Promise<string[]> {
     const branch = this._config.baseBranch;
     const submodules = await this.getSubmodules();
-    const files: string = await this._git.raw([
-      'ls-tree',
-      '-r',
-      '--name-only',
-      branch,
-    ]);
+    const files: string = await this._git.raw(['ls-tree', '-r', branch]);
     // istanbul ignore if
     if (!files) {
       return [];
     }
     return files
       .split('\n')
       .filter(Boolean)
+      .filter((line) => !line.startsWith('120000'))
+      .map((line) => line.split(/\s/).pop())
```

Comment:
```suggestion
      .filter((line) => line.startsWith('100'))
      .map((line) => line.split(/\t/).pop())
```
https://git-scm.com/docs/git-ls-tree#_output_format
https://stackoverflow.com/a/737877/10109857

Reply:
@rarkins We only need regular files, which starts with `100`